chain/chunks/src/lib.rs

```diff
@@ -809,28 +814,31 @@ impl ShardsManager {
         let shard_id = encoded_chunk.header.inner.shard_id;
         let receipts_hashes = self.runtime_adapter.build_receipts_hashes(&receipts).unwrap();
         let (receipts_root, receipts_proofs) = merklize(&receipts_hashes);
-        assert_eq!(encoded_chunk.header.inner.receipts_root, receipts_root);
+        assert_eq!(encoded_chunk.header.inner.outgoing_receipts_root, receipts_root);
         for part_ord in 0..self.runtime_adapter.num_total_parts(&prev_block_hash) {
             let part_ord = part_ord as u64;
             let to_whom = self.runtime_adapter.get_part_owner(&prev_block_hash, part_ord).unwrap();
             let tracking_shards = (0..self.runtime_adapter.num_shards())
-                .filter(|shard_id| {
+                .filter(|shard| {
```

Comment:
same, why change?
scoping works, but if you think it's confusing with variable above - can rename that one to chunk_shard_id

Reply:
chunk_shard_id is awesome